common3.json

```diff
@@ -7419,91 +7419,91 @@
     },
     {
       "id" : 61866,
-      "text" : "Counter:　Enemy side has pokemon with atk > sp.atk"
+      "text" : "Counter: Enemy side has pokemon with atk > sp.atk"
     },
     {
       "id" : 61867,
-      "text" : "Mirror Coat:　Enemy side has pokemon with sp.atk > atk"
+      "text" : "Mirror Coat: Enemy side has pokemon with sp.atk > atk"
     },
     {
       "id" : 61868,
-      "text" : "みやぶる系　対象の回避が2ランク以上アップしている"
+      "text" : "Foresight: Target's evasion is raised by 2 or more stages"
     },
     {
       "id" : 61869,
-      "text" : "みやぶる系　対象がみやぶられ状態"
+      "text" : "Foresight: Target is under foresight"
     },
     {
       "id" : 61870,
-      "text" : "Endure: 自分のＨPが最大ＨPの５割以下"
+      "text" : "Endure: User's HP is less than 50% of max HP"
     },
     {
       "id" : 61871,
-      "text" : "Endure: 味方がじしん系か自爆系の技を持っている"
+      "text" : "Endure: Ally has Earthquate or Self Destruct"
     },
     {
       "id" : 61872,
-      "text" : "Endure: 味方がじしん系か自爆系を選択している"
+      "text" : "Endure: Ally chose Earthquate or Self Destruct"
     },
     {
       "id" : 61873,
-      "text" : "Endure: 既に「Endure」が成功している"
+      "text" : "Endure: Endure has already succeeded"
     },
     {
       "id" : 61874,
-      "text" : "Endure: 「Salac Berry」か「Liechi Berry」を持っていない"
+      "text" : "Endure: Doesn't have Salac Berry or Liechi Berry"
     },
     {
       "id" : 61875,
-      "text" : "Endure: 自分のＨPが1"
+      "text" : "Endure: User's HP is 1"
     },
     {
       "id" : 61876,
-      "text" : "Endure: 天候が「Hail」「砂嵐」である"
+      "text" : "Endure: Weather is Hail or Sandstorm"
     },
     {
       "id" : 61877,
-      "text" : "Curse: 使用者の属性が「Ghost」以外で能力カウント最大値でない"
+      "text" : "Curse: User's type is other than Ghost 能力カウント最大値でない"
```

Comment:
not in your list? (same for the 2 below)

Reply:
I somehow missed these. Translated as well.